OpenRA.Game/ObjectCreator.cs

```diff
@@ -54,7 +55,13 @@ public ObjectCreator(Manifest manifest, FileSystem.FileSystem modFiles)
 				Assembly assembly;
 				if (!ResolvedAssemblies.TryGetValue(hash, out assembly))
 				{
-					assembly = Assembly.Load(data);
+					Stream debugStream = null;
+					if (modFiles.TryOpen(path + ".mdb", out debugStream))
+						assembly = Assembly.Load(data, debugStream.ReadAllBytes());
+					else if (modFiles.TryOpen(path + ".pdb", out debugStream))
```

Comment:
This will not work as expected.

https://github.com/OpenRA/OpenRA/blob/c87e1c789a558d4012c62a5a3cb50e1d175a8ba8/OpenRA.Mods.Common/OpenRA.Mods.Common.csproj#L811-L812